### PR TITLE
Support Deploy SDT via csv

### PR DESCRIPTION
**Pull request by baoy1**
# Description
- Feature: Support Deploy SDT via csv
- Fix: package_details is empty in `powerflex_pacakge` resource
- Fix: occasional package not found issue when deploying cluster via `cluster_resource`

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Manual Test

# Screenshot
Empty package_details 
before fix
empty_package_details
after fix
fix_empty_package_details

### CODE REVIEW

**Comment by terraform-providers-svc** [comment]
Can one of the admins verify this patch?

File: powerflex/provider/cluster_resource.go [hunk 1]
```diff
@@ -319,6 +336,14 @@ func (r *clusterResource) Create(ctx context.Context, req resource.CreateRequest
 		return
 	}
 
+	cookieError := helper.RenewInstallationCookie(r.gatewayClient)
```
Comment (RayLiu7) [inline powerflex/provider/cluster_resource.go:339]: Hi Yiming, could you explain again what the installation cookies are used for?

Comment (baoy1) [reply powerflex/provider/cluster_resource.go:339]: I added the comment for `RenewInstallationCookie`. It is used to resolve "package not found" issue. Please take a look:
```
// RenewInstallationCookie is used to renew the installation cookie, i.e. LEGACYGWCOOKIE.
// Using the same LEGACYGWCOOKIE ensures that the REST requests are sent to the same GW pod.
// That would help to get the correct response from the GW pod that stores installation packages.
```

**Comment by baoy1** [comment]
run e2e test

**Comment by baoy1** [comment]
run e2e test

**Comment by baoy1** [comment]
run e2e test

**Comment by baoy1** [comment]
run e2e test

**Review by RayLiu7 — APPROVED** [review]

**Review by P-Cao — APPROVED** [review]

**Review by shenda1 — APPROVED** [review]

**Pull request merged**